supporting_content/metrics/effectivecost.md

```diff
@@ -1,71 +1,49 @@
 
-###### Note
-*Billed cost is showing the total cash outlay for the reservation. Also note there is no initial Reserved Instance fee representing the cost of the RI for the applicable period.*
-
-
+___
 
-##### Also on each hour a line item is added as follows
+#### Also on each hour a line item is added as follows
 
 | Charge period           | Billing period           | Charge Type | Units | Billed Cost | Effective Cost |
 | ----------------------- | ------------------------ | ----------- | ----- | ----------- | -------------- |
 | Jan 1 2023 - Jan 2 2023 | Jan 1 2023 - Jan 31 2023 | Usage       | 3     | 0           | 3.81369863     |
 
-
-
-###### Note:
-*The amortization is equal to the (Upfront fee / (number of normalized units reserved x periods) x normalized line item units consumed).*
+*Note: The amortization is equal to the (Upfront fee / (number of normalized units reserved x periods) x normalized line item units consumed).*
 
 ___
 
-##### If the Reserved Instance is unused during a period, for example another ineligible instance type is used:
-
+#### If the Reserved Instance is unused during a period, for example another ineligible instance type is used:
 
 | Charge period           | Billing period           | Charge Type     | Units | Billed Cost | Effective Cost |
 | ----------------------- | ------------------------ | --------------- | ----- | ----------- | -------------- |
 | Jan 4 2023 - Jan 5 2023 | Jan 1 2023 - Jan 31 2023 | OD Usag         | 3     | 5.544       | 5.544          |
 | Jan 4 2023 - Jan 5 2023 | Jan 1 2023 - Jan 31 2023 | Unused RI Usage | 3     | 0           | 3.81369863     |
 
+*Note: For the On Demand usage the Billed cost and Effective cost are the same with an Amortization of Zero. Since the RI went completely unused during the period there is an unused RI usage line item with the an amortization equal to (Upfront fee / (number of normalized units reserved x periods) x normalized line item units.*
 
+___
 
-###### Note:
-*For the On Demand usage the Billed cost and Effective cost are the same with an Amortization of Zero. Since the RI went completely unused during the period there is an unused RI usage line item with the an amortization equal to (Upfront fee / (number of normalized units reserved x periods) x normalized line item units.
-
----
-
-##### If the RI is partially unused during a period, for example a scale down occurs:
-
+#### If the RI is partially unused during a period, for example a scale down occurs:
 
 | Charge period           | Billing period           | Charge Type     | Units | Billed Cost | Effective Cost |
 | ----------------------- | ------------------------ | --------------- | ----- | ----------- | -------------- |
 | Jan 4 2023 - Jan 5 2023 | Jan 1 2023 - Jan 31 2023 | OD Usag         | 3     | 5.544       | 5.544          |
 | Jan 4 2023 - Jan 5 2023 | Jan 1 2023 - Jan 31 2023 | Unused RI Usage | 3     | 0           | 3.81369863     |
```

Comment:
```suggestion
| Charge period           | Billing period           | Charge Type     | Units | Billed Cost | Effective Cost |
| ----------------------- | ------------------------ | --------------- | ----- | ----------- | -------------- |
| Jan 4 2023 - Jan 5 2023 | Jan 1 2023 - Jan 31 2023 | OD Usage        | 1     | 0           | 2.542465753          |
| Jan 4 2023 - Jan 5 2023 | Jan 1 2023 - Jan 31 2023 | Unused RI Usage | 2     | 0           | 1.271232877     |
```
Was it intentional that this table is identical to the one above? I believe the intent was to show an example of partially unused RI during a charge period

Reply:
You are right, good catch!